locale/de_DE.UTF-8/LC_MESSAGES/default.po

```diff
@@ -2341,6 +2342,7 @@ msgstr "Eintragen"
 msgid "ended"
 msgstr "vorbei"
 
+# Wie ist dies zu verstehen bitte?
```

Comment:
It means `Engels` need to go to the event so they are marked for being there. They only can apply for shifts (if this mode is enabled) when it is proven that they are at the event.